examples/kitchen-sink/server/controllers/home.js

```diff
@@ -64,6 +65,7 @@ module.exports = (_, response, next) => {
             </section>
           </div>
         </Layout>
+        <ClientEmbed id="custom-context" data={{ foo: true, bar: 'qux' }} />
```

Comment:
In documentation should make it clear to put the embed at the bottom of the page to remove risk of it blocking page render.